eslint.json

```diff
@@ -56,6 +56,7 @@
     "quotes": ["error", "single", "avoid-escape"],
     "semi-spacing": ["error", {"before": false, "after": true}],
     "semi": ["error", "always"],
+    "no-extra-semi": "error",
```

Comment:
I believe we are keeping the options sorted by alphabet, i.e. `no-extra-semi` should be inside the group of other `no-*` options.